src/go/rpk/pkg/cli/cmd/cluster/maintenance/enable.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package maintenance
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newEnableCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "enable <node-id>",
+		Short: "Enable maintenance mode for a node.",
+		Long: `Enable maintenance mode for a node.
+
+This command enables maintenance mode for the node with the specified ID. If a
+node exists that is already in maintenance mode then an error will be returned.
+`,
+		Args: cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			node_id, err := strconv.Atoi(args[0])
```

Comment:
Please change all the snake case names to camel case 😅

Reply:
ouch. fixed